flake.nix

```diff
@@ -30,7 +34,33 @@
               targets = [ "wasm32-unknown-unknown" ];
             })
           ] ++ lib.optionals (stdenv.isLinux) [libudev-zero];
+          postBuild = ''
+            make build-test-wasms
+            '';
+        };
+        stellarcli = stellardev // {
+          cargoLock = {
+            lockFile = ./Cargo.lock;
+          };
+
+          cargoLock.outputHashes = {
+              # The only git+https dependency in Cargo.lock
+              "stellar-rpc-client-21.4.0" = "sha256-ue7Ynux9YaDP3f/XkHz2OPd2g0iCX5R0yS5SaVHEYxQ";
+          };
+
+          doCheck = true;
+
+          GIT_REVISION = "${self.rev or self.dirtyRev or "unknown"}";
+        };
+        rustPlatformMod = pkgs.makeRustPlatform {
+          cargo = pkgs.rust-bin.stable.latest.default;
+          rustc = pkgs.rust-bin.stable.latest.default;
         };
+      in
+      with pkgs;
+      {
+        devShells.default = mkShell stellardev;
+        packages.default = rustPlatformMod.buildRustPackage stellarcli;
       }
```

Comment:
This is outside the scope of the current PR really, but is there a way to specify `--features opt` for the build?